guides/saas-retention-tracking-and-churn-with-powerbi.md

```diff
@@ -0,0 +1,97 @@
+---
+description: Power BI is a unified, scalable platform for self-service and enterprise business intelligence (BI) created and maintained by Microsoft.
+---
+
+# Retention Tracking with Hydra and PowerBI
+
+## Scope
+The focus of this guide is to showcase the ease of integrating Hydra with [PowerBI](https://powerbi.microsoft.com/en-ca/) to develop a customer retention & churn-oriented dashboard on sample data within a Hydra database.
+
+To cater to a broad audience, the data and KPIs were chosen to be as agnostic as possible. The data used is mock sales data created specifically for this demo, and the metrics are the following:
+
+* YoY Time Series Customer Retention Rate by Quarter 
+* YoY Time Series Customer Churn Rate by Quarter
+
+Your KPIs will differ, but this will hopefully serve as a foundation for the development of more metrics.
+
+## Setup
+
+### Before You Start:
+- Ensure you have the details of your Hydra instance handy.
+- Download [PowerBI Desktop](https://powerbi.microsoft.com/en-us/desktop/).
+
+
+### Building the Table with Hydra
+
+1. We will use sample data created specifically for this demo. The data comes in two .sql scripts: one to [create the table](), and one to [populate the table]() with the data. Retreive these two scripts and save them locally.
```

Comment:
Need to add actual links here, you should be able to use relative URLs that will work within the PR and on Gitbook.

Reply:
added